src/styles/GlobalStyles.js

```diff
@@ -0,0 +1,39 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+    :root {
+        --primery-color: #121212;
+        --text-color: #e5e5e5;
+        --running-header-color: #e5e5e5;
+        --default-heder-color: #e5e5e5;
```

Comment:
```suggestion
        --default-header-color: #e5e5e5;
```
Now I know why it worked where you called it 😃